nautobot/ipam/migrations/0030_ipam__namespaces.py

```diff
@@ -132,7 +132,7 @@ class Migration(migrations.Migration):
             model_name="prefix",
             name="namespace",
             field=models.ForeignKey(
-                default=nautobot.ipam.models.get_default_namespace,
+                default=nautobot.ipam.models.get_default_namespace_pk,
```

Comment:
I'm a bit surprised that this change was required seeing as it was absolutely working before.

Reply:
This isn't actually a functional change - I renamed the existing `get_default_namespace` function to `get_default_namespace_pk` for clarity since the new function returns the namespace object and the existing function returns the pk.